Src/FluentAssertions/Equivalency/Node.cs

```diff
@@ -109,14 +130,17 @@ public override bool Equals(object obj)
         return Equals((Node)obj);
     }
 
-    private bool Equals(Node other) => Type == other.Type && PathAndName == other.PathAndName;
+    private bool Equals(Node other) => (Type, Name, Path) == (other.Type, other.Name, other.Path);
```

Comment:
The change from two to three branches is the cause of the remaining decrease in branch coverage.